articles/custom-domains/feature-specific-setup.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Feature Specific Setup with Custom Domains
+description: How to set up custom domains with feature specific instructions
+toc: true
+---
+# Feature-Specific Setup
+
+There are additional steps you must complete depending on which Auth0 features you are using.
+
+::: warning
+If you have been using Auth0 for some time and decide to enable a custom domain, you will have to migrate your existing apps and update the settings as described below. Note that existing sessions created at `tenant.auth0.com` will no longer be valid once you start using your custom domain, so users will have to login again.
+:::
+
+## Configure the Login Page
+
+When using custom domains with [universal login](/hosted-pages/login), you will need to determine which of the following apply to you:
+
+* If you're using the **default** login page without customization, you will not need to make any changes. Your custom domain will work right out of the box.
+* If you're using a **customized** login page, you'll need to update the code in your [Dashboard](${manage_url}) to use your custom domain. The changes that you'll need to make are regarding the initializing of Lock. The following code sample shows the lines reflecting the necessary changes.
+
+```js
+var lock = new Auth0Lock(config.clientID, config.auth0Domain, {
+...
+	configurationBaseUrl: config.clientConfigurationBaseUrl,
+	overrides: {
+		__tenant: config.auth0Tenant,
+		__token_issuer: config.auth0Domain
+	},
+...
+});
+```
+
+## Embedded Lock
+
+If you're using Embedded Lock (Lock v11), you need to use your custom domain when initializing Lock. You will also need to set the **configurationBaseUrl** to the appropriate CDN URL:
```

Comment:
I don't think we should capitalize "Embedded". It is not a feature, it's a way of implementation.